region/maridia/inner-yellow/Plasma Tutorial Room.json

```diff
@@ -141,7 +144,11 @@
         "leaveShinecharged": {
           "framesRemaining": 155
         }
-      }
+      },
+      "unlocksDoors": [
```

Comment:
the g-mode setup above can't use a PB. You can add it or I can later - I should make a full pass through the g-mode strats and check if PBs are ok